### PR TITLE
Nwd nnw

### PR DESCRIPTION
**Pull request by bartosz-gruszczyk**
Solution for the 'NWD-NWW' homework

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: nwd-nnw
🏅 Granted 6 XP for @bartosz-gruszczyk

**Comment by ziobron** [comment]
⏱ 3 XP extra for @bartosz-gruszczyk for on-time delivery

File: homework/nwd-nnw/nwdNww.hpp [hunk 1]
```diff
@@ -1,11 +1,52 @@
...
+
+    if (lhs == 0 && rhs == 0) {
+        return 0;
+    }
+    else if (lhs == 0 && rhs != 0) {
+        return rhs;
+    }
+    else if (rhs == 0 && lhs != 0) {
+        return lhs;
+    }
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:13]: If you return variable here, you don't need to use `else if`

**Review by nauka-programowania-MA — APPROVED** [review]
Perfect! I really enjoy your solutions!

**Review by nauka-programowania-MA — APPROVED** [review]
Perfect! I really enjoy your solutions!

**Pull request closed without merging**